ClassAttendance/src/WebUI/Services/ClassService.cs

```diff
@@ -0,0 +1,139 @@
+﻿using BLL.Interfaces;
+using BLL.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebUI.Models.ViewModels.ClassModel;
+
+namespace WebUI.Services
+{
+    public class ClassService
```

Comment:
Yes, those services acts like facade. I decided to name it "service" because those classes hides all "dirty" work with entities and allows me to call actual service with one 1 line of code. Should i name them with postfix "Facade" or something else? There is no strong naming conventions also. https://softwareengineering.stackexchange.com/a/316842 according to this my naming was right.

Reply:
Yes, but Facade suffix will be more accurate here.

Reply:
Fixed